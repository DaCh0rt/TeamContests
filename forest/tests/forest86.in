857 399
-4119 -1442
6868 5885
-5707 353
854 7656
2010 -1463
2518 4033
3788 -5714
-3839 -4875
-5654 631
7345 -7790
6345 -9060
-8775 -2885
-4470 3470
1210 -1619
-8881 -3564
2953 -1962
-2292 9525
4589 5009
-1997 -2924
126 73
1441 -5950
1938 -8428
-6247 -8342
-1859 -5231
-1050 6209
1913 7321
7136 -1698
-9425 6861
-8401 6505
5614 -9360
1370 -2496
-2334 -3178
-4342 -373
-1888 -5342
-3307 -3228
-2607 8882
5625 -9566
-8321 8198
-6439 -6500
-6987 8541
-8224 1497
-2570 9721
-315 3850
4751 -3331
6424 -7856
-6057 1430
-2324 -1863
2506 -2384
-6321 -3855
1841 8296
5636 8749
4788 1819
6270 -4965
314 -9061
-2101 -8816
9948 7138
-2769 -3312
4984 9660
3309 -2880
9963 107
4890 -5353
8774 -4484
1367 -9725
-2867 6912
-1673 7082
2042 -465
7225 4961
-6457 -4556
-6251 2742
-5507 -9003
8897 2234
-4625 2128
-5670 9141
573 4766
-1298 325
5780 -5840
-5980 6056
-6713 -5144
1620 -4095
-8660 2719
4695 -4642
-519 9643
5249 -3497
-9643 -6509
-7576 -4991
-2804 -7812
8669 1662
-9024 -7125
5676 244
-3319 -7538
-8000 502
-2377 6189
-1938 1810
-9578 -4188
8619 -4072
-9781 -6195
-7475 -7985
-2478 3027
4614 5451
4388 1335
-550 -5115
8011 2146
1953 8838
-1844 2306
-1189 8612
-7801 3229
3359 73
-5034 4533
-9232 -391
-7198 1884
-4675 4463
-8517 -1534
-6648 3276
7770 2954
5287 3770
-9790 551
5730 -3414
3548 3947
3368 1157
52 6471
-9180 1612
-4748 7294
6781 -2025
-766 -3
-1693 6107
-990 -5457
7826 -9541
2243 -4921
-4530 -5115
3355 -1318
5426 -2476
9317 -956
844 -7835
-9630 8476
-3574 -7232
-5651 -2723
553 -8493
-3669 9963
5400 8851
8489 -6483
-5265 3299
-6148 -7928
-8224 7894
-9889 2920
7593 8447
8548 2153
2806 -2466
-9585 -2413
-5890 -2763
2025 7713
-7903 8783
-3132 -4650
-7559 7248
6160 -2381
-6176 -1462
-1395 7083
5754 2853
9908 -4692
-2546 38
8563 1096
8993 -8966
-1351 8061
8161 8686
-4206 4824
-8251 -3682
-7698 2638
6035 1867
1534 6467
1126 -6355
-5084 1363
-2231 -3280
-8033 -9305
1318 -6646
1960 -8984
9360 -2105
358 7995
8250 3104
-5219 -2922
7984 893
7604 -8254
-3746 -9034
-7331 3319
9728 -5321
-5013 -389
-1239 -706
-7885 604
-8369 9200
-9312 180
-2290 3673
1199 4785
-1291 4574
-6519 6309
6081 9859
-8484 -427
115 6839
-2901 -6735
4856 -8850
-6305 -9852
-8770 6870
8267 -9435
-9212 -9804
7316 6448
4958 6415
7369 2507
-3448 -5920
801 5612
-4679 -1127
3517 -4768
-4335 539
-7162 4629
7351 -12
5847 -3338
-97 -5467
-6842 -4111
-6932 -1087
6400 6861
2215 1915
-5276 -8577
-8271 -7497
1727 9567
4043 -81
8687 -4903
6641 7019
3086 -7681
3568 -3106
-2485 -4850
727 -2983
6385 -8648
8175 -602
-5451 -3798
240 4145
5799 -5897
-3644 3107
-7547 -5475
578 -6385
8243 -9224
-7587 -1272
-2674 1018
5596 9220
-234 5511
-1895 -9349
-2904 -482
2493 -7455
-9264 2823
-2998 121
5557 5788
8002 -5401
-3838 -1935
-2390 2002
-9741 -2019
4309 5011
-2354 4425
-2589 -592
2521 6528
9252 204
3281 9223
-7016 4935
-1827 -1817
-1361 364
7490 -9143
-8246 -637
-4019 4574
3194 -9542
6157 5043
-7757 -2819
431 -7128
3847 -1856
-454 -9970
-9226 -6315
-9367 2255
7667 4352
8717 5630
-672 -2026
-5498 -189
-6789 -5639
7659 -8806
-9028 117
6888 9320
4735 -1391
3449 -1973
-4952 696
3396 6519
-96 -9017
646 7364
-6622 -7050
-9423 -3098
-776 4098
-1101 9456
5185 190
-8671 -3754
5080 -5375
-6099 -3452
1618 -4231
-6044 1491
-3049 550
-4586 -17
3396 404
-1021 -9591
-8166 2752
899 -4839
-5087 -1384
-7814 9395
-8865 8017
-1520 9853
7179 -7299
-8447 -3432
-6330 5657
1609 -3698
7087 -8602
2948 -1390
8906 -6063
-4612 -897
-798 9085
7975 -826
-8864 -5952
-592 -7173
-6270 -303
8549 1891
8765 6037
-1178 697
-2996 8722
-2139 4047
-6548 2855
-8207 997
7380 -4318
-9544 -6769
4016 403
9389 3468
3076 4915
1375 -7537
-3153 7051
1130 8720
-2030 -9110
2501 8072
-4485 7155
7634 -7722
-3005 7680
2129 350
4736 -5240
-5388 -2397
1604 -5393
1 -3820
1573 -3726
7142 6311
-4483 7817
-5224 1816
-9903 4678
-2646 6497
1089 4558
943 -1566
1766 -8067
-8142 -5073
8398 4342
4463 -810
6976 7499
3649 2674
-7914 7238
-6592 9755
4538 -5161
-4000 6962
3047 -3370
2781 7805
-3177 -9793
-8202 4719
-1476 4788
-7104 6590
-9800 -4433
8689 -4294
7448 -930
-2354 -1999
2144 1767
-3400 -1871
562 -9956
4098 6470
-3704 4115
3800 2516
1088 4257
6869 -3334
3609 7173
4075 4704
-6155 3374
-7899 256
7938 2829
6456 7569
6336 4569
6235 -4210
-7156 -5448
-9349 6252
-8661 9813
-4741 -6921
-1966 -3060
1521 8634
4120 6109
1651 9159
7037 -7909
-4788 7240
3046 2366
2376 5124
5239 -5226
-6955 -2619
-4309 5195
-3758 -6509
-5665 -5221
-5279 3148
-706 -1868
1329 8883
-8667 -6683
-6251 -7793
-7311 7741
-3768 -3910
1985 -2833
-5433 -3748
-2651 -7672
2334 -3421
-3214 -4289
-7275 -4207
-165 -5458
-2562 -4985
-3492 2104
8341 -2780
-3722 -3438
7433 -8398
-3964 -2969
-4415 -620
-4596 2984
-5719 9441
3762 4559
-3259 -1529
8209 -8478
-4346 -4904
-4641 5476
-2282 5976
3783 -4450
-329 2078
-3298 7177
-854 4963
-2688 -8553
1324 -1349
8876 42
7537 -4339
5140 -1936
-7690 4907
-4342 -5463
9574 5144
1954 181
2871 -7281
-9327 1690
9995 -7823
-7883 -3937
1507 -8236
2783 -9738
9679 1196
8548 -7357
5452 8116
-4003 5807
4551 2921
-6019 -4434
-2120 1676
-5320 -7312
-9737 -8060
-6951 -6245
-1902 5085
-1911 4012
-8095 9081
-3507 912
-279 9432
4278 -7529
-1587 1835
6180 7294
-2368 236
-5451 -8338
2361 2681
-5081 -2460
1355 9523
-4616 2185
-5005 -4511
2229 1100
-4688 7201
4573 8471
-2343 2702
8698 6812
-4769 -4835
9963 -9105
-1667 2003
8994 845
-2108 8209
1632 1256
4883 -725
-5195 8160
-5450 -1833
-7602 -6878
7907 3350
6267 1113
-9356 5510
9072 -423
-1771 -1638
2663 -2996
2457 -1976
-9604 4107
4078 9554
468 -744
-2965 3889
-7403 6014
-5806 9175
-8027 -5911
6518 2015
2719 -8029
-2982 -6215
-7854 2011
306 7473
5760 6129
7992 -7234
-5632 1505
5240 -9699
4172 -4109
5190 -2651
5491 3683
9042 -985
-4195 9824
-8275 9238
9090 -8585
-3933 2123
-645 -9498
4395 -3350
9513 -5980
4409 -6583
2397 968
8695 1243
9373 -2771
1980 -5186
3375 -1504
-7712 889
5558 2758
2534 -3384
353 -6936
3717 -9536
1871 -8316
-6056 6823
7171 7792
-8681 -1748
-7106 6193
9320 -931
9573 956
7021 -6900
-5687 -1969
-7890 8207
-4803 -8570
-2195 155
-579 -1991
-2030 -3458
-7446 8984
-6322 4846
4019 -7171
-6737 5543
1633 -4992
-7964 8637
9039 -2149
1702 -3079
8286 -7756
-2128 -2944
-1020 3185
2353 -392
8280 364
4761 -5182
-6579 -1347
4400 -9912
5080 -2978
7250 4876
-6372 3485
-6625 800
2436 5151
8336 7630
6095 -2555
-7347 795
2377 -82
1874 791
-865 -8860
-5018 6981
1818 -376
-1657 1187
-5050 1883
8137 5370
4798 -3664
-9217 -4434
3308 3670
5993 8759
421 1400
3879 9118
-6758 8596
7595 -8329
-8333 -5785
4822 -3307
-9146 -8232
-526 2408
-3521 4335
-4711 -1132
-8082 -803
2784 -4889
-1712 8607
-6787 -7748
-9423 4363
731 -2485
-9829 -5916
6925 6656
-4132 -9061
1978 359
9811 9430
-8721 -1573
-8727 -1645
5923 4673
-7698 3735
1090 2421
5634 -1411
7797 8131
-2967 9042
-4352 -3277
1035 -3526
8375 4859
6126 -992
817 4071
-4236 1562
-6231 -7468
4779 -3
-4300 -5518
-180 -8477
-8809 8565
-362 2495
9878 -8012
7802 2760
-2973 7927
7114 -2885
381 3775
-5459 8686
-5450 -2495
-9538 -9983
-1578 -9904
2654 -6738
-735 9598
2734 -982
4452 4267
7383 -4020
1469 9549
3270 -3987
1024 -8561
4985 7371
2672 9875
-9128 6921
-2611 -3318
-8201 5913
4022 7895
2658 -3338
4522 8554
8802 -1065
5016 -8281
-372 -8502
-1881 4847
-1219 5434
8566 6596
-2519 4601
4903 -1157
-676 1859
-6911 -7047
27 -5738
5875 7661
-4935 -2616
-3629 5312
-5457 -9002
8750 6087
5205 -7987
-5553 2157
5425 8160
1721 -3825
6810 -764
-5611 17
-7114 139
5814 8540
-9662 7707
-2325 7645
4972 -9537
-724 5289
2019 -7961
3071 -5843
4675 1054
-6506 -2091
9394 -3696
8169 4483
-7962 5519
-5042 1569
-2600 4747
-5630 -456
-7494 -2278
5106 8265
8394 4071
-6839 8081
2541 6441
-3301 5505
-9478 7279
-3337 -3164
135 -9381
9953 -4966
-9141 -9786
-1143 -6733
6172 -7107
9259 -8585
9044 -7773
8811 -3907
419 -2113
-797 6012
-9646 -9125
-9324 5431
736 -3025
5682 3946
-6898 3819
-9605 -4947
9480 -9013
4249 -5260
-2576 890
9209 6661
7312 -304
474 7022
6828 -503
6391 8692
3542 -1136
-7691 -9490
-7649 -3469
5267 5423
-4827 -5627
-935 -7621
-8785 -7830
51 -1520
2066 -5967
-7186 1115
5502 -4423
-7775 -4093
-982 -3103
830 -9556
-454 1218
-6504 -9709
-2174 -1533
-7569 -4070
-4100 -2240
6564 7350
-5795 -2846
2722 -5503
7227 2489
3529 -1285
1975 8953
5072 -1076
234 -9053
-7821 -1789
-6681 353
-7728 1923
-8045 -567
1748 -9336
-6483 167
4387 -5646
-2869 2996
-2216 307
3511 6707
1116 6876
-6765 -2678
-4134 -6030
9771 1278
792 -4261
4645 -3517
-8136 8271
2806 -7104
-9887 -7115
-5562 3181
6982 -3676
7344 -1764
-9813 4189
3264 5967
-2332 6999
9165 -4552
7915 -2132
5166 -5280
2817 7040
-6640 4161
-7367 3639
587 1629
6371 3539
-6379 -2401
4574 437
6834 1721
3391 3655
7087 8768
-2020 9832
952 8733
6749 -1137
-6453 6239
-7292 9157
-2764 -7481
9898 -1388
2432 9354
-5376 -5378
8897 -144
-3899 -5714
-9379 6912
-9479 -5649
7954 -5062
90 5727
1741 6976
989 -6718
3034 -8979
1911 -7167
3117 9871
-5959 -812
-3315 -7906
4207 -3387
-1427 340
-6616 -1626
-6421 1243
1981 -7269
1810 4471
4429 626
-1604 -2273
8258 -4825
4638 -7991
6868 9622
8275 3155
-7780 5
6061 5337
-2298 -3701
5688 -8668
-3862 976
2958 -9035
-3244 -1427
-2556 -1961
-721 6272
4763 -2996
-5743 9452
-5625 -2457
9189 -134
8502 -9543
5692 5047
7400 -2474
-6801 -5037
-9015 1369
5227 -4316
-5688 4181
8859 -5184
-2078 -8560
-1089 -6060
-309 3113
-6977 -7317
7617 1846
-1576 -4658
1055 -2147
1942 3569
-2616 -4644
-7067 4187
-6032 -7871
7601 -7837
-8381 -1683
1132 4374
8850 1876
