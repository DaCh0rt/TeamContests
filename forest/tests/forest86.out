79269.31 420554598.54
